### DISCUSSION TITLE
xrepo 如何查询一个包的所有版本？

### DISCUSSION THREAD

**Discussion by zchrissirhcz** [xmake-io/xmake#5225] [General]
使用官方 xrepo 仓库， 也就是全都是默认配置。

目前只能查询到最新版，例如:
```bash
xmake search opencv
```
结果为:
```
➜  ~ xrepo search opencv
The package names:
    opencv:
      -> opencv-4.10.0: A open source computer vision library. (in xmake-repo)
```

但是我看到 https://github.com/xmake-io/xmake-repo/blob/dev/packages/o/opencv/xmake.lua
这里写了很多版本， 比如 3.4.9, 4.9.0 等。 比如我一个月前的项目用的 opencv 4.9.0， 在新电脑上想安装同样的依赖库， 而不是安装最新的 4.10.0 版本。 在不写 xmake.lua 的情况下， 使用 xrepo 命令能否做到？

**Comment by waruqi** [comment]
```
xrepo install "opencv 4.9.0"
```

**Reply by zchrissirhcz** [reply]
感谢回复，这个方式能安装，不过其实想要的是查询得到可安装版本的列表。

**Reply by waruqi** [reply]
不支持，可以直接到 xmake-repo 仓库翻下，或者到 https://xrepo.xmake.io/ 看
